### DISCUSSION TITLE
invalid choice: 'ins_seg' (choose from 'det', 'track')

### DISCUSSION THREAD

**Discussion by lihongyong** [bdd100k/bdd100k#257] [Q&A]
(yolact) E:\PycharmProjects\yolact>python -m bdd100k.label.to_coco -m ins_seg -i E:/bdd100k_gw/bdd100k/labels/ins_seg/polygons/ins_seg_train.json -o E:/bdd100k_gw/coco_all/annotation.json
usage: to_coco.py [-h] [-i IN_PATH] [-o OUT_PATH] [-m {det,track}] [-ri] [-ic]
to_coco.py: error: argument -m/--mode: invalid choice: 'ins_seg' (choose from 'det', 'track')
--------
help, please!

**Comment by farukcankaya** [comment]
Unfortunately, the authors haven't released a new version of the script on PyPI but updated the documentation. You can install the script from the source of this repository by:
```shell
pip install -e .
```
Then, your command should work!
